### ISSUE TITLE
Question : Is it possible to map a protobuf with snake case field names to Java model which uses camelcase.

### ISSUE THREAD

**Issue by gmishra123** [modelmapper/modelmapper#458]
I am trying to map a protobuf with snake case field names to Java model, but the conversion is failing in the ProtobufValueReader.getMember() call. The member name that this method receives uses snake case, while the Proto generated java class has getters and setters which use camel case and hence an exception is thrown. 
For example my .proto looks like this - 
```
message User{
optional string user_address = 1
}
```
Java model looks like 

```
Class User {
private String userAddress;
}
```
ProtobufValueReader.getMember() method recieves "user_address" which in turn try to invoke getUser_address() method on proto generated java class for reading value and fails at this step. 
This is my configuration - 
```
ModelMapper mapper = new ModelMapper();
mapper.registerModule(new ProtobufModule());
mapper.getConfiguration().setSourceNameTokenizer(NameTokenizers.UNDERSCORE);
mapper.map(userProto,User.class);
```

**Comment by wangzw** [comment]
Same as #422 . I cannot find a way to make it work.

**Comment by gmishra123** [comment]
ok, so I noticed for me If just remove the ```mapper.registerModule(new ProtobufModule());``` from the code things work.

**Comment by chhsiao90** [comment]
I guessed this issue had bean fixed by #466

**Comment by chhsiao90** [comment]
v2.3.4 just release, please take a look, thanks.
